thesis/02-functionality.tex

```diff
@@ -1,27 +1,28 @@
 \subsection{System actors}
 
-Based on the requirements analysis, we have  two types of actors in our system:
+Based on the requirements analysis, we have determined two types of actors in our system:
 
 \begin{description}
 	\item[Folder] \label{actors:folder} - A person who carries out the most important functionality in our system,
-		that is - folds the origami, based on the provided simulation. This person does not need to have any
-		knowledge about origami creation nor the simulation process.
+		that is - folds the origami figure following the provided interactive instruction.
+		This person does not need to have any special knowledge about origami creation nor the simulation process.
 	\item[Designer] \label{actors:designer} - A person who is responsible for providing folding instructions
-		for other users (folders). A Designer is required to have some basic knowledge about origami construction, 
-		and interaction with the system to be able to create and upload a guide. A Designer has to have an account.
+		for other users (\textit{folders}).
+		A Designer is required to have some basic knowledge about origami construction
```

Comment:
>  		A Designer is required to have some basic knowledge about origami construction
->
A designer is required to have a basic understanding of origami construction.